content/en/docs/azure/deploy/install-kubeflow.md

```diff
@@ -145,11 +149,19 @@ Run the following commands to set up and deploy Kubeflow.
       '-', and must start and end with an alphanumeric character.
       The value of this variable cannot be greater than 25 characters. It must
       contain just a name, not a directory path.
-      This value also becomes the name of the directory where your Kubeflow
-      configurations are stored, that is, the Kubeflow application directory.
+      You also use this value as directory name when creating the directory where 
+      your Kubeflow  configurations are stored, that is, the Kubeflow application 
+      directory. 
 
     * **${KF_DIR}** - The full path to your Kubeflow application directory.
 
+    * **${CONFIG_URI}** - The GitHub address of the configuration YAML file that
+      you want to use to deploy Kubeflow. The URI used in this guide is
+      {{% config-uri-k8s-istio %}}.
+      When you run `kfctl apply` or `kfctl build` (see the next step), kfctl creates
+      a local version of the configuration YAML file which you can further
+      customize if necessary.
```

Comment:
Suggestion: for accessibility/a11y, nit

```suggestion
    * **${CONFIG_URI}** - The GitHub address of the configuration YAML file that
      you want to use to deploy Kubeflow. The URI used in this guide is
      {{% config-uri-k8s-istio %}}.
      When you run `kfctl apply` or `kfctl build` (refer to the next step), kfctl creates
      a local version of the configuration YAML file, which you can further
      customize if necessary.
```

Reply:
I literally copied this from https://www.kubeflow.org/docs/started/k8s/kfctl-k8s-istio/ :D